npg_qc_viewer/t/80-app-run.t

```diff
@@ -227,7 +233,83 @@ subtest 'Test for run + lane + plexes' => sub {
+      matches_pf_percent=>'{"33":"0.008749","32":"0.010421","90":"0.009716","63":"0.01368","21":"0.011217","71":"0.012233","7":"0.010641","80":"0.008608","26":"0.009749","18":"0.008166","72":"0.011136","16":"0.009758","44":"0.011065","55":"0.013967","84":"0.007524","74":"0.008031","27":"0.012362","95":"0.011042","57":"0.00938","61":"0.012817","20":"0.008498","92":"0.008779","89":"0.009417","10":"0.0083","31":"0.004355","35":"0.015032","11":"0.009527","91":"0.009209","78":"0.006614","48":"0.000012","87":"0.010499","93":"0.011047","77":"0.010516","65":"0.00816","29":"0.009714","50":"0.013058","39":"0.011873","64":"0.010723","58":"0.008338","41":"0.010025","12":"0.007165","15":"0.009749","81":"0.008173","52":"0.017327","60":"0.013077","56":"0.009526","73":"0.007725","66":"0.011893","45":"0.013335","86":"0.010179","76":"0.008381","19":"0.010513","62":"0.012661","54":"0.015255","67":"0.00839","70":"0.01318","68":"0.011345","17":"0.007324","2":"0.008054","1":"0.006554","88":"0.006101","30":"0.015182","82":"0.008607","25":"0.007871","28":"0.011601","83":"0.009573","75":"0.008364","40":"0.009128","14":"0.009492","69":"0.011525","59":"0.012377","49":"0.006503","24":"0.009485","53":"0.015","79":"0.009684","22":"0.012743","42":"0.010516","0":"0.015999","46":"0.01372","23":"0.010434","13":"0.01012","96":"0.009304","6":"0.010008","85":"0.011683","36":"0.011764","3":"0.009134","94":"0.010348","9":"0.008106","51":"0.010004","47":"0.013507","8":"0.007399","38":"0.014597","4":"0.007356","34":"0.010366","37":"0.013578","43":"0.015139","5":"0.009916"}',
+      max_mismatches_param=>'1',
+      min_mismatch_delta_param=>'1',
+      max_no_calls_param=>'2',
+      pass=>'1',
+      info=>'{"Check":"npg_qc::autoqc::checks::tag_metrics","Check_version":"59.6"}',
+      tag_index=>-1
+    });
+  }
+
+  for my $i (-1 .. 50) {
+    for my $j (6 .. 8) {
+      my $obj = $qc_schema->resultset(q[QXYield])->update_or_new({
+        id_run=>4025,
+        position=>$j,
+        path=>q[some path],
+        filename1=>q[some filename],
+        filename2=>q[other filename],
+        threshold_quality=>'20',
+        yield1=>'1952450',
+        yield2=>'1891079',
+        comments=>'Unrecognised instrument model',
+        info=>'{"Check":"npg_qc::autoqc::checks::qX_yield","Check_version":"59.6"}',
+        tag_index=>$i
+      });
+      if (!$obj->in_storage) {
+        $obj->insert;
+      }
+    }
+  }
+
+  my $url = q[http://localhost/checks/runs?db_lookup=1&run=4025&lane=8&show=all];
+  warning_like{$mech->get_ok($url)} qr/Use of uninitialized value \$id in exists/,
```

Comment:
please exclude this warning test - this is an artifact of the way we authenticate.